css/style.css

```
@@ -0,0 +1,545 @@
+/* Fonts */
+
+@font-face {
+  font-family: "Gilroy";
+  font-display: swap;
+  font-weight: 800;
+  font-style: normal;
+  src: url("../fonts/gilroyextrabold.woff2") format("woff2"),
+       url("../fonts/gilroyextrabold.woff") format("woff");
+}
+
+@font-face {
+  font-family: "Gilroy";
+  font-display: swap;
+  font-weight: 300;
+  font-style: normal;
+  src: url("../fonts/gilroylight.woff2") format("woff2"),
+       url("../fonts/gilroylight.woff") format("woff");
+}
+
+@font-face {
+  font-family: "Open Sans";
+  font-display: swap;
+  font-weight: 400;
+  font-style: normal;
+  src: url("../fonts/opensans.woff2") format("woff2"),
+       url("../fonts/opensans.woff") format("woff");
+}
+
+@font-face {
+  font-family: "Open Sans";
+  font-display: swap;
+  font-weight: 300;
+  font-style: normal;
+  src: url("../fonts/opensanslight.woff2") format("woff2"),
+       url("../fonts/opensanslight.woff") format("woff");
+}
+
+/* Variables */
+
+:root {
+  --basic-black: #000000;
+  --basic-dark: #363636;
+  --basic-white: #FFFFFF;
+  --basic-grey: #464646;
+  --basic-half-grey: #7E7E7E;
+  --basic-grey-light: #E5E5E5;
+  --special-yellow: #F7E296;
+  --special-green: #91C92F;
+  --special-yellow-dark: #F0C52E;
+  --basic-white-light-transparent: rgba(255, 255, 255, 0.6);
+  --basic-white-medium-transparent: rgba(255, 255, 255, 0.3);
+  --basic-white-lighter-transparent: rgba(255, 255, 255, 0.7);
+  --basic-black-light-transparent: rgba(0, 0, 0, 0.6);
+  --basic-black-medium-transparent: rgba(0, 0, 0, 0.3);
+}
+
+/* Global */
+
+.page {
+  height: 100%;
+  padding: 0;
+  margin: 0;
+}
+
+.page__body {
+  min-width: 1180px;
+  min-height: 100%;
+  margin: 0;
+  padding: 0;
+  display: grid;
+  grid-template-rows: min-content 1fr min-content;
+  background-color: var(--basic-white);
+  font-family: "Open Sans", Arial, sans-serif;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.visually-hidden:not(:focus):not(:active),
+input[type="checkbox"].visually-hidden,
+input[type="radio"].visually-hidden {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    margin: -1px;
+    border: 0;
+    padding: 0;
+    white-space: nowrap;
+    clip-path: inset(100%);
+    clip: rect(0 0 0 0);
+    overflow: hidden;
+}
+
+.wrapper {
+  width: 1040px;
+  margin: 0 auto;
+  padding: 0 10px;
```

По [критерию](https://up.htmlacademy.ru/htmlcss/31/criteries#b13) паддинги по 20
```suggestion
  padding: 0 20px;
```

---

тоже сам не понял, как произошло)